### DISCUSSION TITLE
用 ethfs-cli upload 上传图像文件时，遇到 replacement fee too low 错误

### DISCUSSION THREAD

**Discussion by linghuccc** [IntensiveCoLearning/Web3-URL#206] [Q&A] [answered]
我尝试用 ethfs-cli upload 上传一个 1 MB 的图像文件到 Sepolia chain，但是运行到 chunk 7, 8, 9 就停住了，所以我杀了进程，重新上传了一遍，结果就遇到了下面的错误：

![error](https://github.com/user-attachments/assets/64cfa31e-469e-46f1-a410-b65df99aac54)

求教这是什么原因，以及应该怎么办？

**Comment by qzhodl** [comment]
看样子是因为有一笔交易没有确认，卡在那里。可以发一个转账交易，增加gas price，把现在卡住的交易replace掉

**Reply by linghuccc** [reply]
需要手动创建一个相同 nonce ID 的转账交易，不能用 metamask 发起，是吧？

**Reply by qzhodl** [reply]
metamask可以指定nonce

**Reply by qzhodl** [reply, marked as answer]
在 settings => advanced 里面启用这个feature就行

**Reply by qzhodl** [reply]
在替换的时候，需要提高gas price，才能替换成功

**Reply by linghuccc** [reply]
我尝试了用 metamask，无法成功。又写了 python 代码，但是执行的时候会报错：
![debug](https://github.com/user-attachments/assets/9eabd362-271a-4c6b-a0ea-51ac8c8ea60b)

**Reply by qzhodl** [reply]
gas price 提高了吗？

**Reply by linghuccc** [reply]
我放的 20 Gwei，但还是一样不行。可能当时 Sepolia 网络有问题，我现在看，已经上传成功了。
